src/mumble/MainWindow.cpp

```diff
@@ -899,43 +899,172 @@ void MainWindow::openUrl(const QUrl &url) {
 	g.sh->start(QThread::TimeCriticalPriority);
 }
 
+class ChannelUrlPathHandler {
```

Comment:
I put the stuff into a helper class with static methods, in cpp file scope.
Maybe I should put this into a separate file?